lib/routes/app_router.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:zachranobed/routes/app_router.gr.dart';
+import 'package:zachranobed/routes/guards/auth_guard.dart';
+
+@AutoRouterConfig(replaceInRouteName: 'Screen,Route')
+class AppRouter extends $AppRouter {
+  @override
+  List<MaterialRoute> get routes => [
+        MaterialRoute(
+          page: HomeRoute.page,
+          initial: true,
+          guards: [AuthGuard()],
+        ),
+        MaterialRoute(page: LoginRoute.page),
```

Comment:
Jediná věc, ale to asi do budoucna. Když user otevře appku a nebude na některé jiné routě, thankyou apod. a mezitím appka ho odhlasí, tak ty ho z té appky nevyhodíš, protože máš guard jen na / . Ale for now asi nevadí 👍

Reply:
Přidáno i k ostatním routám